src/components/Home/Home.jsx

```diff
@@ -1,7 +1,10 @@
+import Navigation from "../Navigation";
+
 const Home = () => {
     return ( 
         <div>
-            Home
+            <Navigation/>
```

Comment:
The above  omments means you can leave the navigation component out of the home component